docker/todo.dockerfile

```diff
@@ -13,7 +13,11 @@ COPY --from=composer /var/www/vendor/ vendor
 
 RUN docker-php-ext-install pdo pdo_mysql
 RUN pecl install xdebug && docker-php-ext-enable xdebug
+
 ENV ENV .todo.env.docker
+ENV DB_HOST=mysql
```

Comment:
i moved these out of .todo.env.docker so I can change them for the fly.io instances.  Typically you don't want to store this stuff directly in git 😬 .  But this is fine for testing. Note for the fly.io instances I changed the password to something else.